client/app/services/query-snippet.js

```diff
@@ -1,8 +1,12 @@
-export let QuerySnippet = null; // eslint-disable-line import/no-mutable-exports
+import axios from "@/services/axios";
+import { extend, map } from "lodash";
 
-function QuerySnippetService($resource) {
-  const resource = $resource("api/query_snippets/:id", { id: "@id" });
-  resource.prototype.getSnippet = function getSnippet() {
+class QuerySnippet {
```

Comment:
Example of simple class in a service (this one is read-only, so has no issues).

I could rewrite the `getSnippet` in a non-class way, but wanted to show a simple class in here (for other services it's the same base idea, but a lot more complex)